create-llama.sh

```diff
@@ -20,7 +20,7 @@ npx -y create-llama@0.1.25 \
     --post-install-action none \
     --no-llama-parse \
     --example-file \
-    --vector-db none \
+    --vector-db milvus \
```

Comment:
goal: we just need to change this from `none` to `milvus` to use Milvus in chat llamaindex